cleverhans/attacks_tfe.py

```diff
@@ -8,7 +8,8 @@
 
 from cleverhans import attacks
 from cleverhans import utils
-from cleverhans.model import CallableModelWrapper
+from cleverhans.compat import reduce_sum
```

Comment:
Is this new import a typo?

Reply:
I had to rebase to open the PR. When I resolved the merge conflict, I thought that import had been added in the master branch. Now I have no idea where it came from. Deleting it.